### ISSUE TITLE
FreeBSD: AttributeError: 'module' object has no attribute 'ETIME'

### ISSUE THREAD

**Issue by alxwr** [RaetProtocol/raet#11]
FreeBSD(10)'s Python2.7 module 'errno' does not define ETIME

```python
>>> import errno
>>> errno.ETIME                                                                       
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'ETIME'
>>> errno.EHOSTDOWN
```

```bash
$ uname -a
FreeBSD cloud 10.3-RELEASE-p11 FreeBSD 10.3-RELEASE-p11
```

```bash
$ python2.7 --version
Python 2.7.12
```

Got it working this way:

```diff
--- /usr/local/lib/python2.7/site-packages/raet/stacking.py.orig        2016-12-19 18:39:59.183409232 +0100
+++ /usr/local/lib/python2.7/site-packages/raet/stacking.py     2016-12-19 18:44:24.693570735 +0100
@@ -408,10 +408,13 @@
             self.server.send(tx, ta)
         except socket.error as ex:
             err = raeting.get_exception_error(ex)
-            if (err in [errno.EAGAIN, errno.EWOULDBLOCK,
-                             errno.ENETUNREACH, errno.ETIME,
-                             errno.EHOSTUNREACH, errno.EHOSTDOWN,
-                             errno.ECONNRESET]):
+            errors = [errno.EAGAIN, errno.EWOULDBLOCK,
+                      errno.ENETUNREACH,
+                      errno.EHOSTUNREACH, errno.EHOSTDOWN,
+                      errno.ECONNRESET]
+            if hasattr(errno, 'ETIME'):
+                errors.append(errno.ETIME)
+            if (err in errors):
                 # problem sending such as busy with last message. save it for later
                 laters.append((tx, ta))
                 blocks.append(ta)
```

**Comment by SmithSamuelM** [comment]
Thanks,  
I will make the fix.

Interesting that Darwin's FreeBSD has it but not the Standard FreeBSD

**Comment by alxwr** [comment]
Interesting indeed. :-)

Thank you for the quick response!

**Comment by SmithSamuelM** [comment]
Fixed see 40a4c6097c3f715de24841c668ceb0a506c166be

**Comment by alxwr** [comment]
Thank you!
